client/js/analysis.js

```diff
@@ -317,3 +322,13 @@ function showLightbox() {
   target.classList.remove('hide-lightbox');
 }
 
+/**
+ * Shakes the search bar and make it glow
+ */
```

Comment:
small nit: add that this is for errors
`Shakes the search bar and make it glow for user error`